Cython/Compiler/ModuleNode.py

```diff
@@ -3097,11 +3097,7 @@ def generate_type_import_call(self, type, code, import_generator, error_code=Non
         if sizeof_objstruct != objstruct:
             if not condition:
                 code.putln("")  # start in new line
-            code.putln("#if defined(PYPY_VERSION_NUM) && PYPY_VERSION_NUM < 0x050B0000")
```

Comment:
Doesn't this just remove backwards compatibility with PyPy 5 (however good or bad that was)?